content/getting-started/multiDeviceMultiSig/_index.en.md

```diff
@@ -0,0 +1,174 @@
+
+<img src = laptopWalletConfig.png
+    alt = "wallet configuration (Screenshot)">
+
+4. You can now assign the key stored locally on your laptop. Notice that the other two keys are considered foreign and your local key is recognized as a "SOFTWARE" key.<br/><br/>
+
+<img src = laptopAssignKey.png
+    alt = "assign key configuration (Screenshot)">
+
+5. You should now be returned to the chat screen and see that your wallet is now only pending 1 key.<br/><br/>
+
+<img src = laptopPendingKeys.png
+    alt = "pending keys (Screenshot)">
+
+6. Continue to your next device!
+
+{{< /tab >}}
+
+{{< tab "Mobile" >}}
+
+{{< notice note >}}
+If you did not have all of your keys available on your desktop when assigning keys to wallet, you can continue the assignment from another device. It is assumed that you have started to create a wallet and can see a pending Wallet Creation notification in the chat screen of your laptop. If you do not, you should go back to the desktop tab and create a wallet first.
+{{< /notice >}}
+
+1. Open your Nunchuk mobile app and navigate to the chat screen. You should see your pending Wallet Creation notification here waiting for the last key!<br/><br/>
+
+<img src = mobilePendingKey.png
+    alt = "mobile app chat screen (Screenshot)">
+
+2. Click on the "View config" button.<br/><br/>
+
+3. Click the "Continue to assign keys" button at the bottom of the screen.<br/><br/>
+
+<img src = mobileContinueAssignKeys.png
+    alt = "Wallet configuration (Screenshot)">
+
+4. You can now assign the last key (in the screenshot below, it is the Mobile Key).<br/><br/>
+
+5. Click the "Assign keys" to finalize your wallet with the last key.<br/><br/>
+
+<img src = mobileAssignKeysSendInvite.png
+    alt = "assign keys and send invite (Screenshot)">
+
+6. You should now be returned to the chat screen and see that the wallet status has changed to "Pending finalization". On any of your devices you can open the wallet config again by clicking the "View Config" and then the "Finalize wallet" button!<br/><br/>
+
+<img src = mobileAllKeysAssigned.png
+    alt = "mobile phone chat screen (Screenshot)">
+
+7. YReview the wallet configuration one more time, make sure everything is correct, then click the "Finalize wallet" button.<br/><br/>
+
+<img src = desktopFinalizeWalletConfig.png
+    alt = "desktop wallet config (Screenshot)">
+
+8. You're all done! You should see your wallet on the main screen.<br/><br/>
+
+{{< /tab >}}
+{{</ tabs >}}
+
+{{< notice tip >}}
+It is highly recommended to back up your wallet (noted in red, below). There are 2 options for recovery. Either (a) the BSMS file AND enough keys to satisfy the number of signatures requirement or (b) all keys and brute-forcing to "guess" what the original BSMS file was (this can fail with highly-customized and more advanced wallet configurations).If you do not have a backup or a working wallet, you risk losing access to your funds forever.
```

Comment:
let's move this tip to the very bottom (after the "Typically for collaborative wallets..." paragraph)